exercises/handling-get-requests/views/home.erb

```diff
@@ -10,10 +10,26 @@
 <% end %>
 
 <h3>Search!</h3>
-<p>Wish there was a search field...</p>
+<p>Wow look a form for search!</p>
+
+<form action="/search" method = "get">
```

Comment:
In HTML, you wouldn't include spaces around the `=` symbol when defining attributes in a tag.